tool/tctl/common/resource_command.go

```diff
@@ -583,6 +585,38 @@ func (rc *ResourceCommand) createSessionRecordingConfig(ctx context.Context, cli
 	return nil
 }
 
+// createExternalAudit implements `tctl create external_audit.yaml` command.
+func (rc *ResourceCommand) createExternalAudit(ctx context.Context, client auth.ClientI, raw services.UnknownResource) error {
+	config, err := services.UnmarshalExternalAudit(raw.Raw)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	externalAuditClient := client.ExternalAuditClient()
+	_, err = externalAuditClient.CreateExternalAudit(ctx, config)
+	if err != nil {
+		return trace.Wrap(err)
+	}
```

Comment:
The outer code will display "use `tctl create -f`" on an `AlreadyExists` error, which won't help here. Maybe we can just return the error as a generic `trace.Errorf` if it's `AlreadyExists`?

Reply:
I reworked it to be upsert, so it won't apply anymore.